heat/core/statistics.py

```diff
@@ -376,6 +377,72 @@ def average(x, axis=None, weights=None, returned=False):
     return result
 
 
+def bincount(x, weights=None, minlength: int = 0):
+    """
+    Count number of occurrences of each value in array of non-negative ints.
+
+    The number of bins (size 1) is one larger than the largest value in `x`
+    unless `x` is empty, in which case the result is a tensor of size 0.
+    If `minlength` is specified, the number of bins is at least `minlength` and
+    if `x` is empty, then the result is tensor of size `minlength` filled with zeros.
+    If `n` is the value at position `i`, `out[n] += weights[i]` if weights is specified else `out[n] += 1`.
+
+    Parameters
+    ----------
+    x : DNDarray, 1 dimensional, non-negative ints
+    weights : DNDarray, optional
+        Weight for each value in the input tensor. Array of the same shape as x. Same split as `x`.
+    minlength : int, non-negative, optional
+        Minimum number of bins
+
+    Returns
+    -------
+    out : DNDArray
```

Comment:
Maybe mention that out will have `split=None` in the docs.

Reply:
added